infor-design/enterprise
# 2695 - Fix info and alert icons on tabs and inside of modal

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
The info and alert icons on tabs and inside of modal doesn't align properly the way the error icon does. Some CSS rules inside of the tabs affects the icons that was not in modal. Moving some css rules to the modal as a parent instead.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/2695

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app

1.)
- Navigate to http://localhost:4000/components/tabs/test-modal-error.html
- Click `Add Context` button
- Tab out in input field to trigger the validation error
- Error icon will display on tab section
- Inspect the element of the icon
- Change the class `icon-error` to `icon-info`, then after `icon-alert`
- Both `icon-info` and `icon alert` should align properly
- Test also on uplift theme

2.)
- Navigate to http://localhost:4000/components/tabs/example-dropdown-tabs.html
- Click on `Paper` tab, then click the `Plates` to show the data that has input field
- Trigger the validation error by tabbing out
- Error icon will show on tab
- Inspect the element of the icon
- Change the class `icon-error` to `icon-info`, then after `icon-alert`
- Both `icon-info` and `icon alert` should align properly
- Try also to trigger the focus of tab
- Icon should have more space and not close to the border tab

Additional Test at 2.)
- Inspect the element of sub list of tab that has icon when validation trigger. See image below
<img width="100" alt="err-icon" src="https://user-images.githubusercontent.com/8839327/80954241-986b5700-8e2f-11ea-8d19-7181ac278b93.png">

- Change the class `icon-error` to `icon-info`, then after `icon-alert`
- Both `icon-info` and `icon alert` should align properly
- Test this also on uplift theme

**Included in this Pull Request**:
